paasta_tools/cli/cmds/secret.py

```diff
@@ -224,17 +236,37 @@ def _add_vault_auth_args(parser: argparse.ArgumentParser):
     )
 
 
-def _add_common_args(parser: argparse.ArgumentParser, allow_shared: bool = True):
-    # available from any subcommand
+def _add_vault_auth_args_for_add_and_update(parser: argparse.ArgumentParser):
     parser.add_argument(
         "-y",
         "--yelpsoa-config-root",
         dest="yelpsoa_config_root",
         help="A directory from which yelpsoa-configs should be read from",
-        default=DEFAULT_SOA_DIR,
+        default=os.getcwd(),
```

Comment:
Any opinion on using this or "./"?

Reply:
imo, getcwd() is a little nicer aesthetically (and makes it more obvious that this is what we want rather than someone seeing `./` and wondering if maybe there was a typo before reading the rest of the code)